mmv1/products/dialogflowcx/Webhook.yaml

```diff
@@ -71,27 +83,134 @@ properties:
+          Specifies a list of allowed custom CA certificates (in DER format) for
+          HTTPS verification. This overrides the default SSL trust store. If this
+          is empty or unspecified, Dialogflow will use Google's default trust store
+          to verify certificates.
+          N.B. Make sure the HTTPS server certificates are signed with "subject alt
+          name". For instance a certificate can be self-signed using the following
+          command,
+          openssl x509 -req -days 200 -in example.com.csr \
+          -signkey example.com.key \
+          -out example.com.crt \
+          -extfile <(printf "\nsubjectAltName='DNS:www.example.com'")
         item_type:
           type: String
+      - name: httpMethod
+        type: Enum
+        description: |-
+          HTTP method for the flexible webhook calls. Standard webhook always uses
+          POST.
+        enum_values:
+          - 'POST'
+          - 'GET'
+          - 'HEAD'
+          - 'PUT'
+          - 'DELETE'
+          - 'PATCH'
+          - 'OPTIONS'
+      - name: oauthConfig
+        type: NestedObject
+        description: |-
+          Represents configuration of OAuth client credential flow for 3rd party
+          API authentication.
+        properties:
+          - name: clientId
+            type: String
+            description: The client ID provided by the 3rd party platform.
+            required: true
+          - name: clientSecret
+            type: String
+            description: |-
+              The client secret provided by the 3rd party platform.  If the
+              `secret_version_for_client_secret` field is set, this field will be
+              ignored.
+            ignore_read: true
+          - name: scopes
+            type: Array
+            description: The OAuth scopes to grant.
+            item_type:
+              type: String
+          - name: secretVersionForClientSecret
+            type: String
+            description: |-
+              The name of the SecretManager secret version resource storing the
+              client secret. If this field is set, the `client_secret` field will be
+              ignored.
+              Format: `projects/{project}/secrets/{secret}/versions/{version}`
+          - name: tokenEndpoint
+            type: String
+            description: |-
+              The token endpoint provided by the 3rd party platform to exchange an
+              access token.
+            required: true
+      - name: parameterMapping
+        type: KeyValuePairs
+        description: |-
+          Maps the values extracted from specific fields of the flexible webhook
+          response into session parameters.
+          - Key: session parameter name
+          - Value: field path in the webhook response
+      - name: requestBody
+        type: String
+        description: Defines a custom JSON object as request body to send to flexible webhook.
+      - name: requestHeaders
```

Comment:
Just want to make sure, this field looks to be immutable previously, is removing `immutable` the right move?